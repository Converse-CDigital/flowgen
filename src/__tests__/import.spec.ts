// @flow

import { compiler, beautify } from "..";

it("should handle dynamic imports", () => {
  const ts = `
type A = import('react');
type B = import('react').ReactNode;
`;
  expect(
    beautify(compiler.compileDefinitionString(ts, { quiet: true })),
  ).toMatchSnapshot();
});
