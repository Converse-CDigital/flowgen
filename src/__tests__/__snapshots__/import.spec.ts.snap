// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle dynamic imports 1`] = `
"declare type A = $Exports<\\"react\\">;
declare type B = $PropertyType<$Exports<\\"react\\">, \\"ReactNode\\">;
"
`;
